manifold/specs/logspout.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: logspout
+  namespace: gliderlabs
+spec:
+  template:
+    metadata:
+      name: logspout
+      labels:
+        app: logspout
+    spec:
+      containers:
+      - image: gliderlabs/logspout:v3.1
+        imagePullPolicy: Always
+        name: logspout
+        args: ["syslog+tls://logs5.papertrailapp.com:31048"]
```

Comment:
It's just this one place, pretty clear from filename where it might be. I say leave it.